ccd-definition/FixedLists/HearingVenue.json

```diff
@@ -3,609 +3,616 @@
     "LiveFrom": "01/01/2017",
     "ID": "HearingVenue",
     "ListElementCode": "1",
-    "ListElement": "CARDIFF ABERDARE HEARING CENTRE",
+    "ListElement": "Cardiff, Aberdare Hearing Centre",
     "DisplayOrder": 1
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "HearingVenue",
     "ListElementCode": "2",
-    "ListElement": "CARDIFF BLACKWOOD HEARING CENTRE NP2 2XB",
+    "ListElement": "Cardiff, Blackwood Hearing Centre, Np2 2Xb",
```

Comment:
Not sure if we should lowercase the postcode what do BAs/PO think?

Reply:
thanks @akomisarek, I have just asked for confirmation from the BAs.

Reply:
@akomisarek,[confirmed with BAs and updated.